sdk/core/core-http/lib/util/utils.ts

```diff
@@ -98,8 +105,13 @@ export function objectValues(obj: { [key: string]: any; }): any[] {
       }
     }
   } else {
-    throw new Error(`The provided object ${JSON.stringify(obj, undefined, 2)} is not a valid object that can be ` +
-      `enumerated to provide its values as an array.`);
+    throw new Error(
+      `The provided object ${JSON.stringify(
+        obj,
+        undefined,
+        2
+      )} is not a valid object that can be ` + `enumerated to provide its values as an array.`
```

Comment:
Oh, this PR is meant to only fix formatting issues.
Lets do refactoring for readability after the end of main implementation (Atom management API)

Reply:
Yes, I understand this PR is to fix the formatting so that further PRs would not have the noise and I appreciate the effort.

As you know, formatting tools exist to make it easier for us to maintain coding styles across all files. If at any point, they cause more harm than good, then we should try our best to fix that especially if the fix is very trivial.

This is one such case.

I would prefer to have this fixed rather than have this PR introduce a new problem and then have another PR to fix it.

Also, there is a high chance that the implementation for the management apis might not touch this line of code and you might want that PR to be restricted to code related to that feature.